### ISSUE TITLE
Header toolbar doesn't align all the elements in one line

### ISSUE THREAD

**Issue by georgique** [fullcalendar/fullcalendar#6216]
### Reduced Test Case

https://codesandbox.io/s/fc5-angular-toolbar-alignment-bug-iqdp2

### Bug Description

In certain configurations of the toolbar, items are not aligned in one line. Reason is that toolbar itself is a flexbox container, which has items: left, center and right. However, those items themselves are containers too, and they don't handle multiple children correctly.
Browser: FF 87 / Chrome 89
Free Calendar version: 5.5.0 [Angular]

### Screenshots

![image](https://user-images.githubusercontent.com/15987211/112690522-2336b500-8e39-11eb-8f56-8a7add162f1a.png)

**Comment by acerix** [comment]
When I try your demo, all the buttons show on one line, unless I make the window too small to fit them.

**Comment by georgique** [comment]
You are right. My bad, didn't try the test case with wider screen.
However, title is still a problem:
![image](https://user-images.githubusercontent.com/15987211/112880241-0ab7dc00-907f-11eb-8aff-9d506b8f069e.png)

Problem with "today" button I've figured out - it was my theme spoiling the styles. I'll deal with it.

**Comment by acerix** [comment]
Yes, it seems like a bug that the title is not on the same line, I get the same without Angular:

https://codepen.io/acerix/pen/VwPKMXX?editors=0110

As a workaround that can be fixed with CSS:

https://codepen.io/acerix/pen/QWdKOyN?editors=0110
